WebRoot/messages/ZMsg_iw.properties

```diff
@@ -39,18 +39,18 @@ account.PASSWORD_LOCKED = \u05d4\u05e1\u05d9\u05e1\u05de\u05d4 \u05e0\u05e2\u05d
 account.PASSWORD_RECENTLY_USED = \u05d4\u05e1\u05d9\u05e1\u05de\u05d4 \u05e9\u05e9\u05dc\u05d7\u05ea \u05d4\u05d9\u05d9\u05ea\u05d4 \u05d1\u05e9\u05d9\u05de\u05d5\u05e9 \u05dc\u05d0\u05d7\u05e8\u05d5\u05e0\u05d4. \u05d1\u05d7\u05e8 \u05e1\u05d9\u05e1\u05de\u05d4 \u05d0\u05d7\u05e8\u05ea.
 
 # {0} = filename
-formatter.INVALID_FORMAT = \u05ea\u05d1\u05e0\u05d9\u05ea \u05e7\u05d5\u05d1\u05e5 \u05dc\u05d0-\u05d7\u05d5\u05e7\u05d9\u05ea \u05e2\u05d1\u05d5\u05e8 \u05d4\u05e7\u05d5\u05d1\u05e5 '{0}'.
+formatter.INVALID_FORMAT = \u05ea\u05d1\u05e0\u05d9\u05ea \u05e7\u05d5\u05d1\u05e5 \u05dc\u05d0-\u05d7\u05d5\u05e7\u05d9\u05ea \u05e2\u05d1\u05d5\u05e8 \u05d4\u05e7\u05d5\u05d1\u05e5 ''{0}''.

```

Comment:
@mihaipopahtp No problem. I'd be happy to take a look and review the changes. Another set of eyes (and code) on it to spot discrepancies wouldn't hurt I think ;-) . 
@silentsakky I agree that tackling this specific issue on a global level should be a better solution than a piecemeal approach with a handful of pull requests.
Thanks to both for the feedback.